Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -376,3 +377,40 @@ class AppStoreConnectService {
         }
     }
 }
+
+extension AppStoreConnectService {
+
+    func getBuildIdAndGroupIdsFrom(
+        bundleId: String,
+        version: String,
+        buildNumber: String,
+        groupNames: [String]
+    ) throws -> (buildId: String, groupIds: [String]) {
+        let appId = try ReadAppOperation(options: .init(identifier: .bundleId(bundleId)))
+            .execute(with: requestor)
+            .await()
+            .id
+
+        let buildId = try ReadBuildOperation(
+                options: .init(
+                    appId: appId,
+                    buildNumber: buildNumber,
+                    preReleaseVersion: version
+                )
+            )
+            .execute(with: requestor)
+            .await()
+            .build
+            .id
+
+        let groupIds = try groupNames.flatMap {
+            try ListBetaGroupsOperation(options: .init(appIds: [appId], names: [$0]))
+                .execute(with: requestor)
+                .await()
+                .map(\.betaGroup.id)
+        }
```

Comment:
Why not something like:
```suggestion
        let groupIds = try ListBetaGroupsOperation(options: .init(appIds: [appId], names: groupNames))
                .execute(with: requestor)
                .await()
                .map(\.betaGroup.id)
```

I feel we could make a single network request here rather than many

Reply:
After some tries found API will aggregate all the filters so input two filters at a time: 
`name1 && name2` can find a group called ->  `name1name2` 
Seems multiple requests is required when searching with multiple group names 🤔

Reply:
I feel like with n group names it might be less requests to load all groups for a single app and then filter in the tool...
Let's try that